python/create_client/__init__.py

```diff
@@ -21,12 +21,12 @@
 
 def is_create_running(sg_connection=None):
     """
-    Launch Shotgun Create and inject the current authentication informations into the Shotgun Create session.
+    Checks for the status of Shotgun Create.
 
-    :param Shotgun sg_connection: Shotgun connection to use with the CreateClient. If not set, the connection
-    from the current bundle is used.
+    :param Shotgun sg_connection: Shotgun connection to use with the CreateClient. \
+    If not set, the connection from the current bundle is used.
```

Comment:
```suggestion
        If not set, the connection from the current bundle is used.
```
Just indent, no need to use `\`